common/namespace/namespace.go

```diff
@@ -290,3 +309,27 @@ func (n Name) String() string {
 func (n Name) IsEmpty() bool {
 	return n == EmptyName
 }
+
+func (m *CustomSearchAttributesMapper) GetAlias(fieldName string, namespace string) (string, error) {
```

Comment:
Consider using a type alias here so that callers don't suffer from type blindness since "fieldName" and "alias" are both strings but have very different meanings